addons/web_editor/static/src/js/frontend/loader.js

```diff
@@ -76,6 +80,11 @@ async function loadFromTextarea(parent, textarea, options) {
         $textarea.val(await wysiwyg.getValue());
     });
 
+    // Remove the loading spinner.
+    if (loading) {
+        loading.parentNode.removeChild(loading);
```

Comment:
I have a hard time figuring out whether it is still loading or not. I think the `loading` variable should be called `spinner`.

Reply:
Yeah I'm not a fan either but I copied the code that had "disappeared" from 14.0 and figured I should probably not change it if I didn't _have_ to. But I can rename it anyway if you think it's ok for us to do so.

Reply:
I didn't know this was code that has "disappeared" :wink: :wink: It's ok then. Could you maybe rebuild JW on this PR tomorrow and ping me on discord when you do it ? In this way I am "blessing" this PR as being the next build merger.